### ISSUE TITLE
v1.8.1: schxslt:report in SVRL

### ISSUE THREAD

**Issue by AirQuick** [schxslt/schxslt#212]
Compiling `https://github.com/xspec/xspec/raw/v2.1.4/test/schematron/schematron-01.sch` with SchXslt v1.8.1 and then validating `https://github.com/xspec/xspec/raw/v2.1.4/test/schematron/schematron-01.xml` with the compiled stylesheet generates this `svrl:schematron-output` which contains `schxslt:report`:

```xml
<svrl:schematron-output ...>
   <svrl:metadata ...>
   ...
   </svrl:metadata>
   <schxslt:report>
      <schxslt:document>
         <schxslt:pattern id="d12e2"
                          documents="https://raw.githubusercontent.com/xspec/xspec/v2.1.4/test/schematron/schematron-01.xml">
            <svrl:active-pattern documents="https://raw.githubusercontent.com/xspec/xspec/v2.1.4/test/schematron/schematron-01.xml"/>
         </schxslt:pattern>
      </schxslt:document>
      <schxslt:rule pattern="d12e2">
         <svrl:fired-rule context="article"/>
      </schxslt:rule>
      <schxslt:rule pattern="d12e2">
         <svrl:fired-rule context="section"/>
      </schxslt:rule>
      <schxslt:rule pattern="d12e2">
         <svrl:fired-rule context="section"/>
         <svrl:failed-assert location="/Q{}article[1]/Q{}section[2]" id="a002" test="title">
            <svrl:text>
                section should have a title
            </svrl:text>
         </svrl:failed-assert>
      </schxslt:rule>
      <schxslt:rule pattern="d12e2">
         <svrl:fired-rule context="section"/>
         <svrl:failed-assert location="/Q{}article[1]/Q{}section[3]" id="a002" test="title">
            <svrl:text>
                section should have a title
            </svrl:text>
         </svrl:failed-assert>
      </schxslt:rule>
   </schxslt:report>
   <svrl:active-pattern documents="https://raw.githubusercontent.com/xspec/xspec/v2.1.4/test/schematron/schematron-01.xml"/>
</svrl:schematron-output>
```

Is it working as intended?

On v1.8, the same files generate no `schxslt:report`:

```xml
<svrl:schematron-output ...>
   <svrl:metadata ...>
    ...
   </svrl:metadata>
   <svrl:active-pattern documents="https://raw.githubusercontent.com/xspec/xspec/v2.1.4/test/schematron/schematron-01.xml"/>
   <svrl:fired-rule context="article"/>
   <svrl:fired-rule context="section"/>
   <svrl:fired-rule context="section"/>
   <svrl:failed-assert location="/Q{}article[1]/Q{}section[2]" id="a002" test="title">
      <svrl:text>
                section should have a title
            </svrl:text>
   </svrl:failed-assert>
   <svrl:fired-rule context="section"/>
   <svrl:failed-assert location="/Q{}article[1]/Q{}section[3]" id="a002" test="title">
      <svrl:text>
                section should have a title
            </svrl:text>
   </svrl:failed-assert>
</svrl:schematron-output>
```

**Comment by dmj** [comment]
Oh dear. No, this is an error on my side. Will be fixed in 1.8.2

**Comment by AirQuick** [comment]
Thanks for the fix. v1.8.2 looks good concerning XSpec's test set.
